### PR TITLE
samples/tests style cleanups.

### PR DESCRIPTION
**Pull request by dahliamalkhi**
This PR addresses Michael's review on #400 , and also a few cleanups of tests and samples.

### CODE REVIEW

File: runtime/src/main/resources/logback.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         </encoder>
     </appender>
 
-    <root level="ERROR">
+    <root level="DEBUG">
```
Comment (no2chem) [inline runtime/src/main/resources/logback.xml]: Is there a reason you changed the default log level?

File: test/src/test/java/org/corfudb/runtime/object/transactions/WriteAfterWriteTransactionContextTest.java [hunk 2]
```diff
@@ -10,14 +10,25 @@
...
     /** In a write after write transaction, concurrent modifications
      * with the same read timestamp should abort.
      */
     @Test
     public void concurrentModificationsCauseAbort()
     {
+        getMap();
         t(1, () -> write("k" , "v1"));
-        t(1, this::TXBegin);
+        t(1, () -> TXBegin());
```
Comment (no2chem) [inline test/src/test/java/org/corfudb/runtime/object/transactions/WriteAfterWriteTransactionContextTest.java]: why the convention change here?

File: test/src/test/resources/logback-test.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
         </encoder>
     </appender>
 
-    <root level="TRACE">
-        <!--<appender-ref ref="STDOUT" />-->
+    <root level="DEBUG">
```
Comment (no2chem) [inline test/src/test/resources/logback-test.xml]: same as previous comment.

File: test/src/test/java/org/corfudb/runtime/object/transactions/TXConflictScenarios.java [hunk 4]
```diff
@@ -259,13 +254,16 @@ public void finegrainedUpdatesDoNotConflict()
...
-                .build()
-                .setStreamName("test stream")
-                .setSerializer(Serializers.JSON)
-                .addOption(ObjectOpenOptions.NO_CACHE)
-                .setTypeToken(new TypeToken<SMRMap<String, String>>() {})
-                .open();
+        Map<String, String> testMap2 = (Map<String, String>)
+                instantiateCorfuObject(
+                        new TypeToken<SMRMap<String, String>>() {}, "test stream");
+//        getRuntime().getObjectsView()
```
Comment (no2chem) [inline test/src/test/java/org/corfudb/runtime/object/transactions/TXConflictScenarios.java]: seems like you should just remove these comments?

File: runtime/src/main/resources/logback.xml (hunk 1 above)
Comment (dahliamalkhi) [inline runtime/src/main/resources/logback.xml]: leftover. fixed.

File: test/src/test/java/org/corfudb/runtime/object/transactions/WriteAfterWriteTransactionContextTest.java (hunk 2 above)
Comment (dahliamalkhi) [inline test/src/test/java/org/corfudb/runtime/object/transactions/WriteAfterWriteTransactionContextTest.java]: seems cleaner, not a big deal.

File: test/src/test/resources/logback-test.xml (hunk 3 above)
Comment (dahliamalkhi) [inline test/src/test/resources/logback-test.xml]: done.

File: test/src/test/java/org/corfudb/runtime/object/transactions/TXConflictScenarios.java (hunk 4 above)
Comment (dahliamalkhi) [inline test/src/test/java/org/corfudb/runtime/object/transactions/TXConflictScenarios.java]: done.

**Pull request merged**